### ISSUE TITLE
Blurred background in the DMG (macOS installer)

### ISSUE THREAD

**Issue by zrzka** [balena-io/etcher#2643]
See attached image. Repo contains `background.png` and `background@2x.png`, but the DMG contains TIFF (`background.tiff`), which contains just the standard resolution version (not both). And it leads to a blurred background image. Compare _balenaEtcher.app_ & _Applications_ with _An open source project by balena_.

<img width="833" alt="screenshot 2019-02-22 at 13 35 37" src="https://user-images.githubusercontent.com/1084172/53243102-1f0e9b00-36a7-11e9-9252-561eae251a03.png">

**Comment by lurch** [comment]
I've no idea what `tiffutil` does, and I don't have a Mac myself, but the `Makefile` _seems_ to be implying that `background.tiff` is made from both `background.png` and `background@2x.png` ?
https://github.com/balena-io/etcher/blob/master/Makefile#L108

**Comment by zrzka** [comment]
`tiffutil` in the `Makefile` is supposed to build one `.tiff` image from both `background.png` (standard) and `background@2x.png` (hidpi) images. But the repository already contains `background.tiff` with the same date & time and thus `make assets/dmg/background.tiff` does nothing:

```sh
(master {1} ✓) etcher make assets/dmg/background.tiff 
make: `assets/dmg/background.tiff' is up to date.
(master {1} ✓) etcher
```

And the `background.tiff` in the repo doesn't contain hidpi version, just the standard one.

<img width="870" alt="screenshot 2019-02-23 at 00 32 12" src="https://user-images.githubusercontent.com/1084172/53277185-f3b99980-3702-11e9-991b-ba544fd1ffea.png">

To update it, one has to touch either file:

```
(master {1} ✓) etcher touch assets/dmg/background.png 
(master {1} ✓) etcher make assets/dmg/background.tiff 
tiffutil -cathidpicheck assets/dmg/background.png assets/dmg/background@2x.png -out assets/dmg/background.tiff
2 images written to assets/dmg/background.tiff.
(master {1} U:1 ✗) etcher
```

And now, the `background.tiff` contains both versions.

<img width="870" alt="screenshot 2019-02-23 at 00 33 14" src="https://user-images.githubusercontent.com/1084172/53277223-25cafb80-3703-11e9-8a4a-131c4e8f463c.png">

**Comment by lurch** [comment]
I guess this can be closed, it seems it was included in #2674 ?
